src/module/helpers/missingModule.js

```diff
@@ -18,7 +19,7 @@ export function isMissingModule (store, moduleName) {
  * @param {String} moduleName
  */
 export function registerMissingModule (store, api, moduleName) {
-  const builder = new ModuleBuilder(store, api, moduleName, {}, { standalone: true })
+  const builder = new ModuleBuilder(store, api, moduleName, new ResourceProxy(), { standalone: true })
```

Comment:
This would make requests on a missing module of wander off god knows where. You're right that this is a bug though. Might the solution be: 
```suggestion
  const builder = new ModuleBuilder(store, api, moduleName, api[moduleName] ?? new ResourceProxy(), { standalone: true })
```

This would fetch the proxy from the api if available. Still not satisfying but hopefully closer to correct.